test/Spec/Memory/insert.spec.luau

```diff
@@ -73,8 +73,9 @@ return function()
 		local returnedDestroy, returnedDestroy2 = insert(scope, onDestroy, onDestroy2)
 		expect(returnedDestroy).to.equal(onDestroy)
 		expect(returnedDestroy2).to.equal(onDestroy2)
-		expect(#scope).to.equal(2)
-		doCleanup(scope)
-		expect(counter).to.equal(3)
+		print("AQUI!")
+		-- expect(#scope).to.equal(2)
+		-- doCleanup(scope)
+		-- expect(counter).to.equal(3)
```

Comment:
Is there a reason this section was replaced with a print statement?
Or was it just left over from debugging?

Reply:
Oh yeah that's a mistake. Good catch